pkgs/servers/sql/postgresql/default.nix

```
@@ -24,6 +24,10 @@ let
       , nukeReferences, patchelf, llvmPackages
       , makeRustPlatform, buildPgxExtension, cargo, rustc
 
+      # PAM
+      , pamSupport ? false
+      , linux-pam
+
```

```suggestion
      , linux-pam
```
since linux-pam is a dependency of systemd already, we can move this just below line 7 to the other dependencies and remove the condition competently or make it just dependend on stdenv.isLinux